deploy/playbooks/roles/common/tasks/main.yml

```diff
@@ -1,52 +1,41 @@
 ---
-- name: "Build hosts file"
-  sudo: yes
-  template:
-    src: hosts.j2
-    dest: "/etc/hosts"
-
-- name: Set Hostname with hostname command
```

Comment:
I think this and the /etc/hosts task above were to help with rabbitmq issues. You need to ensure that you have a set hostname before you start rabbitmq, because if you start rabbitmq with one hostname, change the hostname on the system when you restart rabbitmq it fails. Did we find another way around this?